env.js

```diff
@@ -113,6 +113,7 @@ function setStorage() {
   env.profile_collection = readENV('MONGO_PROFILE_COLLECTION', 'profile');
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
   env.food_collection = readENV('MONGO_FOOD_COLLECTION', 'food');
+  env.azurepush_collection = readENV('MONGO_AZUREPUSH_COLLECTION', 'azurepush');
```

Comment:
I've been wanted a generic notifications collection, can we rename this to `notifications` instead of `azurepush` then later we can do some refactoring to be able to use it for other notification integrations.  For example the pushover receipts are currently only stored in memory, would be nice to be able to add them to a collection to survive a restart.